src/main/java/com/exactpro/th2/common/schema/box/configuration/BoxConfiguration.java

```diff
@@ -38,11 +41,15 @@ public void setBoxName(@Nullable String boxName) {
         this.boxName = boxName;
     }
 
+    @NotNull
     public String getBookName() {
         return bookName;
     }
 
-    public void setBookName(String bookName) {
+    public void setBookName(@NotNull String bookName) {
+        if (isBlank(bookName)) {
+            throw new IllegalArgumentException("Book name cannot be null or blank");
+        }
```

Comment:
Use the requireNonBlankBookName method

Reply:
Thanks, I missed it.
Also I change `requireNonNullParentId()`/`requireNonBlankBookName()`/`requireNonBlankScope()` to public